lib/install/validate/validator.go

```diff
@@ -205,6 +205,20 @@ func (v *Validator) basics(ctx context.Context, input *data.Data, conf *config.V
 	conf.SetName(input.DisplayName)
 	conf.SetDebug(input.Debug.Debug)
 	conf.Name = input.DisplayName
+
+	// Set default size of scratch disk
+	if input.ScratchSize == "" {
+		log.Warnln("Using default scratch size of 8GB")
+		input.ScratchSize = "8G"
+	}
+	scratchSize := parseScratchSize(input.ScratchSize)
+	if scratchSize < 0 { // TODO set minimum size of scratch disk
+		v.NoteIssue(errors.Errorf("Invalid default image size %s provided", input.ScratchSize))
+	} else {
+		conf.ScratchSize = scratchSize
```

Comment:
I disagree we need to validate free space.  Our disks are sparse.  It is possible to have 8 8GB disks on a 1GB datastore and happily run as long as the data written didn't exceed the datastore size.  This qualifies as a quota check and that's outside this change and our intent.